Update sprint2_1.md (#85)
bSI-InfraRoom/IFC-Tunnel-Deployment: sprints/sprint2_1.md

    - All `IfcSite` shall be aggregated in `IfcProject`.
- [Spatial Decomposition](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/spatial-decomposition.htm)
    - Each `IfcTunnel` may only be decomposed by `IfcTunnel` or `IfcTunnelPart`.
    - There shall be at least one `IfcTunnel` in the file with non-empty `Name`.
- Linear Composition
    - in progress: https://github.com/bSI-InfraRoom/IFC-Specification/issues/523
    - There shall be one `IfcAlignment` in the file with non-empty `Name`.
- [Alignment Layout](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/alignment-layout.htm)
    - There shall be at least one `LINE`, at least one `CIRCULARARC` and at least one `CLOTHOID` segments nested in the horizontal layout (see `IfcAlignmentHorizontalSegment.PredefinedType`).
    - There shall be at least one `CONSTANTGRADIENT` and either at least one `CIRCULARARC` or at least one `PARABOLICARC` segments nested in the vertical layout (see `IfcAlignmentVerticalSegment.PredefinedType`).
- [Object Nesting](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/object-nesting.htm)

>     - There shall be at least one `IfcReferent` in the file with non-empty `Name`.

Reviewer: I don't like to see these rules. Not possible to check. Propose:
```suggestion
    - There shall be at least one `IfcReferent` on `IfcAlignment` with non-empty `Name`.
```
